chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -163,7 +163,7 @@ metadata:
   name: compass-connector-one-time-tokens
   namespace: {{ .Release.Namespace }}
 spec:
-  description: Configuration of oathkeeper for secure endpoint of compass gateway
+  # Configuration of oathkeeper for secure endpoint of compass gateway
```

Comment:
Having "description" seem to be still valid syntax.
https://github.com/ory/oathkeeper-maester/blob/master/config/samples/oathkeeper_v1alpha1_rule.yaml#L13

Reply:
"description" seems to miss here:
https://github.com/ory/oathkeeper-maester/blob/master/api/v1alpha1/rule_types.go#L59
Is it possible this example to be outdated?